### PR TITLE
New formula libpointing-binary

### PR DESCRIPTION
**Pull request by libpointing**
This is the binary version of the libpointing Formula. Binary version has platform specific functionality and research data.
Libpointing is a software toolkit that provides direct access to HID pointing devices and supports the design and evaluation of pointing transfer functions.

The toolkit provides resolution and frequency information for the available pointing and display devices and makes it easy to choose between them at run-time through the use of URIs. It allows to bypass the system's transfer functions to receive raw asynchronous events from one or more pointing devices.

### CODE REVIEW

File: libpointing-binary.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+class LibpointingBinary < Formula
+  homepage "http://pointing.org"
+  url "http://libpointing.org/homebrew/libpointing-binary-0.92.tar.gz"
```
Comment (mistydemeo) [inline libpointing-binary.rb]: What OS X versions and CPU architectures are these binaries compatible with?

**Comment by mistydemeo** [comment]
The extra functionality in this binary can't be built from source?

**Comment by mistydemeo** [comment]
@BrewTestBot test this please

File: libpointing-binary.rb (hunk 1 above)
Comment (libpointing) [inline libpointing-binary.rb]: It is compatible with 10.10 Yosemite

**Comment by libpointing** [comment]
No, it is not possible to build it from source. We cannot show source codes for this version.

File: libpointing-binary.rb (hunk 1 above)
Comment (libpointing) [inline libpointing-binary.rb]: How can we restrain it to work with only Yosemite for now?

Comment (libpointing) [inline libpointing-binary.rb]: Now it is compatible with the minimum macosx version 10.6. Please merge the pull request.

File: libpointing-binary.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
+class LibpointingBinary < Formula
+  homepage "http://pointing.org"
```
Comment (mikemcquaid) [inline libpointing-binary.rb]: This homepage doesn't say anything about this project.

Comment (libpointing) [inline libpointing-binary.rb]: Finally, thank you for pointing this out. I will correct it write now.

Comment (libpointing) [inline libpointing-binary.rb]: Done. The homepage was corrected.

**Comment by mistydemeo** [comment]
Is there a more specific name for this than `libpointing-binary`? I'm concerned that users might not understand what the distinction is.

**Comment by libpointing** [comment]
libpointing-binary appears to be the most suitable name for this formula as it only contains the compiled version of the library together with the headers. We will clearly explain that on the libpointing website.

**Comment by mikemcquaid** [comment]
> libpointing-binary appears to be the most suitable name for this formula as it only contains the compiled version of the library together with the headers. We will clearly explain that on the libpointing website.

I think we'd like to see that before this gets included (and more details on the license).

**Comment by mistydemeo** [comment]
Is there not a name that more clearly distinguishes what the different functionality is? There's already a `libpointing` formula in core, so I'd rather see a name that gives a sense of what the difference is - not just the info on the libpointing site.

**Comment by bfontaine** [comment]
Any update here?

**Comment by libpointing** [comment]
Hello, could you please close this pull request. No need to merge it. I have created a separate tap. And could you please remove "libpointing" formula from the main tap.

**Pull request closed without merging**